draft-ietf-quic-qpack.md

```diff
@@ -550,18 +550,36 @@ the block.  Blocking decoders use the Largest Reference to determine when it is
 safe to process the rest of the block.
 
 `Base Index` is used to resolve references in the dynamic table as described in
-{{indexing}}.  To save space, Base Index is encoded relative to Largest
-Reference using a one-bit sign flag.
+{{indexing}}.
 
-    baseIndex = largestReference + deltaBaseIndex
+To save space, Base Index is encoded as a relative to the Base Index using a
+one-bit sign and the `Delta Base Index` value.  A sign bit set to 0 indicates
+that the Base Index has a greater absolute index than the Largest Reference; the
+value of Delta Base Index is added to the Largest Reference to determine the
+absolute value of the Base Index.  A sign bit of 1 indicates that the Base Index
+is smaller than the Base Index.
 
-If the encoder inserted entries to the table while the encoding the block,
-Largest Reference will be greater than Base Index, so deltaBaseIndex will be
-negative and encoded with S=1.  If the block did not reference the most recent
-entry in the table and did not insert any new entries, Largest Reference will be
-less than Base Index, so deltaBaseIndex will be positive and encoded with S=0.
-When Largest Reference and Base Index are equal, deltaBaseIndex is 0 and encoded
-with S=0.
+That is, after turning a sign bit of 1 to -1 and 0 to 1, the absolute Base Index
+is:
+
+~~~
+    baseIndex = largestReference + sign * deltaBaseIndex
```

Comment:
While order of operations makes this correct, it might be worth parentheses for clarity.